client/my-sites/site-settings/site-settings-form-fieldset/index.jsx

```diff
@@ -0,0 +1,20 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import FormFieldset from 'components/forms/form-fieldset';
+import InfoPopover from 'components/info-popover';
+
+export const SiteSettingsFormFieldset = ( { isJetpackSite, children, className } ) => {
+	if ( ! isJetpackSite ) {
+		return null;
```

Comment:
We might want to display a placeholder instead of nothing. See https://github.com/Automattic/wp-calypso/blob/master/docs/reactivity.md